sdk/servicebus/azservicebus/main_test.go

```diff
@@ -0,0 +1,15 @@
+package azservicebus
+
+import (
+	"os"
+	"testing"
+
+	"github.com/joho/godotenv"
+)
+
+func TestMain(m *testing.M) {
+	godotenv.Load()
```

Comment:
I saw in the other file you had to include the file, is that optional or required?

Reply:
Basically, if omitted it just assumes that the .env file is in the current folder.

With that said, I should test that it worked properly. I know the root folder one (this one) does.

Reply:
Okay, just tested it out a few ways and it's working the way I want it (in that you only have to have a single .env file, at the 'root' of the azservicebus package to run the tests.

There was some other code I got rid of that was attempting to load the .env file as well, so those are all gone and now it's pure TestMain().